project.godot

```diff
@@ -18,12 +18,13 @@ config/icon="res://icon.png"
 
 global_player="*res://global_player.gd"
 Base="*res://Base.gd"
+global_menu="*res://global_menu.gd"
 
 [display]
 
 window/dpi/allow_hidpi=true
 window/stretch/mode="2d"
-window/stretch/aspect="keep"
+window/stretch/aspect="expand"
```

Comment:
Did you push the rebase?

Reply:
yeah

Reply:
hmmm, why it be saying this should be change